src/main/java/gg/amy/catnip/utilities/waiter/EventExtension.java

```diff
@@ -0,0 +1,20 @@
+package gg.amy.catnip.utilities.waiter;
+
+import com.mewna.catnip.extension.AbstractExtension;
+import com.mewna.catnip.shard.EventType;
+import lombok.experimental.Accessors;
+
+/**
+ * @author AdrianTodt
+ * @since 1/15/18.
+ */
+@Accessors(fluent = true)
```

Comment:
Don't need to have this unless you actually have any accessors in the class

Reply:
done 👍